### ISSUE TITLE
Fix for build/util/LASTCHANGE 

### ISSUE THREAD

**Issue by meniossin** [kiwibrowser/src#76]
When you're debugging remotely and trying to inspect(devtools console) a page/url from inside the chrome://inspect, you will get a 404(not found). 

This issue can be found [here](https://github.com/kiwibrowser/src#remote-debugging) .

The LASTCHANGE is actually invalid and refers to a non-existing commit. thus the 404 error when loading the devtools console.

**Comment by meniossin** [comment]
After trying to solve this issue, these are steps I followed and the results I came about :

1. Replaced the versionInfo var in build/util/lastchange.py with a recent commit from the master branch of chromium(found [here](https://chromium.googlesource.com/chromium/src/)).

2.Also, after replacing the version commit, I removed the indicated files as suggested [here](https://github.com/kiwibrowser/src#remote-debugging).

3. Rebuilded the project, re-installed the apk in my device and inspected the device via the browser.
The error of 404 for the devtools console still persists nonetheless.
